test/integration/serverless/serverless.yml

```diff
@@ -11,14 +11,11 @@ provider:
     lambda: true
     apiGateway: true
   environment:
-    DD_INTEGRATION_TEST: true
```

Comment:
This env var causes dd-lambda-js to print all HTTP requests to logs. It doesn't do anything in the extension or other Lambda Libraries. We don't want this behavior in the extension integration tests so I removed the env var.